config/merge_shadowlands.lua

```diff
@@ -31,52 +31,67 @@ end
 header "|cffd2d3d8SL|r™ |cff798BDDItems|r"
 do
 end
+--]]
 
-header "|cffd2d3d8SL|r™ |cff798BDDWorld Zones|r"
+header "|cffd2d3d8SL|r™ |cff798BDDQuest Items|r"
 do
 end
 
-header "|cffd2d3d8SL|r™ |cff798BDDQuest Items|r"
+header "|cffd2d3d8SL|r™ |cff798BDDWorld Zones|r"
 do
+	item '327909' '0.5' "breaking ashen phylactery" -- Thorgast
 end
---]]
+
 header "|cffd2d3d8ShadowLands|r™ |cff798BDDCovenants|r"
 do
-	-- Venthyr
 	-- warlock
-	item '321792' '1.0' "Impending Catastrophe"
+	alias '325640' '234153' --  Drain life multi with soulrot
```

Comment:
The id being aliased here is already defined as an item below, so not sure that this works as expected? All soul rot damage will show as drain life damage?

Reply:
Mistake from me :) 
Sorry for late reply i was playing wow since i was very busy with work :)
As for the soul rot, i do see the damages and healling in differents ticks, but i actually never checked :S